wolfhsm/wh_server.h

```diff
@@ -58,8 +58,10 @@ typedef struct {
     int devId;
     Aes aes[1];
     RsaKey rsa[1];
+#ifdef HAVE_ECC
```

Comment:
When are we building without ECC? Maybe the Aes (NO_AES), RsaKey (NO_RSA) and curve_25519 (HAVE_CURVE25519) should also be added?